components/button/floating-buttons.js

```diff
@@ -15,12 +15,7 @@ class FloatingButtons extends RtlMixin(LitElement) {
 			/**
 			 * Indicates to display buttons as always floating
 			 */
-			alwaysFloat: { type: Boolean, attribute: 'always-float' },
-
-			/**
-			 * The minimum height of the viewport to display floating buttons at (where applicable)
-			 */
-			minHeight: { type: String, attribute: 'min-height' },
+			alwaysFloat: { type: Boolean, attribute: 'always-float', reflect: true },
```

Comment:
Noticed that the CSS has an attribute selector for `always-float` so this property needs to reflect.